src/ocm/components/clusterConfiguration/CpuArchitectureDropdown.tsx

```diff
@@ -44,43 +58,53 @@ const getInvalidCombinationReason = (
     : undefined;
 };
 
-const CpuArchitectureDropdown = () => {
+type CpuArchitectureDropdownProps = {
+  openshiftVersion: Cluster['openshiftVersion'];
+  day1CpuArchitecture?: CpuArchitecture;
+  cpuArchitectures: SupportLevels;
```

Comment:
I still find confusing to name `SupportLevels` data as `cpuArchitectures`.
PLus this is an object than then has a field named `architectures`